src/vuestic-theme/vuestic-sass/global/_content.scss

```diff
@@ -0,0 +1,74 @@
+
+.content {
+  h1 {
+    @include va-display(1);
+    @extend .mt-0;
```

Comment:
That's just bootstrap override. Also `!important` is very hard to override.

We may want to mark this handling explicitly as well as point in another task to get rid of it after we're done with bootstrap.